datadog.conf.example

```diff
@@ -153,6 +153,9 @@ gce_updated_hostname: yes
 #  Make sure your client is sending to the same port.
 # dogstatsd_port: 8125
 
+# buffer size options in Ks -> Default is 8. Choose from 16, 32 or 64. 
+# buffer_size: 8
```

Comment:
We should be clearer with this name. Which buffer is this configuring? What are the implications of changing it?